cmake/set_up_platform.cmake

```diff
@@ -30,27 +30,30 @@ macro(set_up_platform)
 
   include(GNUInstallDirs)
 
-  if (EKAT_INCLUDE_DIR)
-    if (NOT EXISTS ${EKAT_INCLUDE_DIR})
-      message(FATAL_ERROR "Couldn't find EKAT include dir at ${EKAT_INCLUDE_DIR}.")
+  # Configure EKAT (whether built externally or internally).
+  if (EKAT_SOURCE_DIR)
+    if (NOT EXISTS ${EKAT_SOURCE_DIR})
+      message(FATAL_ERROR "Invalid EKAT source dir: ${EKAT_SOURCE_DIR}.")
+    elseif()
+      if (NOT EXISTS ${EKAT_SOURCE_DIR}/src)
+        message(FATAL_ERROR "EKAT source dir has no src/ ѕubdirectory!")
```

Comment:
How about adding the full name of the missing directory to the error message in this and the error messages below.  Might make it easier to debug.